### DISCUSSION TITLE
magic-tape: a tui yt client/downloader, with image support (including chafa)

### DISCUSSION THREAD

**Discussion by christosangelopoulos** [hpjansson/chafa#226] [Show and tell]
Thanks for the awesome work.

I have written a bash script called [magic-tape](https://gitlab.com/christosangel/magic-tape).

It a youtube client/downloader, with a few image support protocols for the video thumbnails, chafa being one of them. 

![chafa_preview](https://github.com/user-attachments/assets/d94c6601-930f-4aa6-bee8-96ad7c0d0c90)

I would like to let you know of it, expressing again my thanks for your good effort.

**Comment by hpjansson** [comment]
That looks really useful!

**Reply by christosangelopoulos** [reply]
Well, I sure like it and use it every day, but then again I wrote it, so I am not objective! I don't use a browser for yt anymore.

I also have written something else in c using chafa, I am still testing. When I publish it, I will surely let you know.

Thanks again, keep up the good work.

**Reply by hpjansson** [reply]
Yes, absolutely let me know. I'll add it to the app list. Thanks.

**Reply by christosangelopoulos** [reply]
Thank you, chafa rocks.
